docs/reference/commandline/trust_signer_remove.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "signer-remove"
+description: "The signer-remove command description and usage"
+keywords: "signer, notary, trust"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# trust signer-remove
+
+```markdown
+Usage:  docker trust signer-remove [OPTIONS] signer IMAGE [IMAGE...]
+
+Remove a signer from an image
+
+```
+
+## Description
+
+`docker trust signer-remove` removes signers from signed repositories.
+
+## Examples
+
+### Remove a signer from a repo
+
+To remove an existing signer, `alice`, from this image: 
+
+```bash
+$ docker trust inspect example/trust-demo
+
+No signatures for example/trust-demo
+
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+alice               05e87edcaecb
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo:
+Repository Key:	ecc457614c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4555b3c6ab02f71e
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+
+We can remove `alice` with `docker trust signer-remove`:
+
+```bash
+$ docker trust signer-remove alice example/trust-demo
+  Enter passphrase for repository key with ID 642692c: 
+  Successfully removed alice from example/trust-demo
+
+```
+
+`docker trust inspect` now does not list `alice` as a valid signer:
+
+```bash
+$ docker trust inspect example/trust-demo
+
+No signatures for example/trust-demo
+
+
+List of signers and their KeyIDs:
+
+SIGNER              KEYS
+bob                 5600f5ab76a2
+
+Administrative keys for example/trust-demo:
+Repository Key:	ecc457614c9fc399da523a5f4e24fe306a0a6ee1cc79a10e4555b3c6ab02f71e
+Root Key:	3cb2228f6561e58f46dbc4cda4fcaff9d5ef22e865a94636f82450d1d2234949
+```
+
+### Remove a signer from multiple repos
+
+To remove an existing signer, `alice`, from multiple images: 
```

Comment:
for both `signer-add` and `signer-remove`: could we add an example of multiple-image add/remove with interspersed successes and failures?

It's important for us to highlight that these commands are best effort, and that the operation might partially succeed

Reply:
Do other docs show failures? I couldn't find an example of that.

Reply:
I don't think any of our `docker trust` ones do so far, but I think this case is important to highlight